src/test/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtilsUnitTest.java

```diff
@@ -70,7 +70,7 @@ private VariantContext makeVC(String source, List<Allele> alleles, Genotype... g
     }
 
     private VariantContext makeVC(String source, List<Allele> alleles, String filter) {
-        return makeVC(source, alleles, filter.equals(".") ? null : new LinkedHashSet<>(Collections.singletonList(filter)));
+        return makeVC(source, alleles, filter.equals(".") ? null : new LinkedHashSet<String>(Collections.singletonList(filter)));
```

Comment:
@LeeTL1220 maybe your intellij isn't set up to use JDK7+ features? You should be able to fix this on the Project Settings page.